codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -359,6 +359,10 @@ private Names() {
     @Deprecated
     public static final class Values {
         /**
+         * {@code "application/json"}
+         */
+        public static final String APPLICATION_JSON = "application/json";
```

Comment:
actually we shouldn't add new entries to a deprecated class ... let me remove this